test/functional/bundleadd/add-json.bats

```diff
@@ -24,61 +24,67 @@ test_setup() {
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 41, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 45, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 50, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 54, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 58, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 62, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 66, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 70, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 75, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 79, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 83, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 87, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 91, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 95, "stepDescription" : "install_files" \},
-		\{ "type" : "progress", "currentStep" : 6, "totalSteps" : 7, "stepCompletion" : 100, "stepDescription" : "install_files" \},
-		\{ "type" : "info", "msg" : "Calling post-update helper scripts. " \},
-		\{ "type" : "progress", "currentStep" : 7, "totalSteps" : 7, "stepCompletion" : 100, "stepDescription" : "run_scripts" \},
-		\{ "type" : "info", "msg" : "Successfully installed 1 bundle " \},
-		\{ "type" : "end", "section" : "bundle-add", "status" : 0 \}
-		\]
+		[
+		{ "type" : "start", "section" : "bundle-add" },
+		{ "type" : "progress", "currentStep" : 1, "totalSteps" : 7, "stepCompletion" : 100, "stepDescription" : "load_manifests" },
+		{ "type" : "progress", "currentStep" : 2, "totalSteps" : 7, "stepCompletion" : 100, "stepDescription" : "consolidate_files" },
+		{ "type" : "progress", "currentStep" : 3, "totalSteps" : 7, "stepCompletion" : 100, "stepDescription" : "check_disk_space_availability" },
+		{ "type" : "info", "msg" : "Downloading packs (0.13 Mb) for: " },
+		{ "type" : "info", "msg" : " - test-bundle " },
+		{ "type" : "progress", "currentStep" : 4, "totalSteps" : 7, "stepCompletion" : 12, "stepDescription" : "download_packs" },
```

Comment:
it's very very very hard to maintain this test. I'd use regexp to check only for the things we are really interested on.

Reply:
This problem was fixed in a latter commit.